templates/docs/examples/patterns/pricing-block/block-with-normal-rule.html

```diff
@@ -0,0 +1,68 @@
+{% extends '_layouts/examples.html' %}
+{% from '_macros/vf_pricing-block.jinja' import vf_pricing_block %}
+{% block title %}Pricing block / With optional elements{% endblock %}
```

Comment:
Title needs to be updated
```suggestion
{% block title %}Pricing block / With normal rule{% endblock %}
```